Rubberduck.Refactorings/RefactoringsUI.resx

```diff
@@ -619,4 +621,86 @@ Do you want to proceed?</value>
     <value>Select the new Type of Declaration Statement for '{0}'.</value>
     <comment>0: Selected target</comment>
   </data>
+  <data name="ExtractMethod_CompilationDirectivesWarning" xml:space="preserve">
+    <value>Selected code contains compiler directives. Extracting this code may cause breaking changes.</value>
+  </data>
+  <data name="ExtractMethod_ReturnTypeSelection" xml:space="preserve">
+    <value>Select variable to use as return value of new function. Select None to make a sub.</value>
+  </data>
+  <data name="ExtractMethod_CodePreviewCaption" xml:space="preserve">
+    <value>Preview of method extracted from '{0}'</value>
+    <comment>0: Source method</comment>
+  </data>
+  <data name="ExtractMethod_DefaultNewMethodName" xml:space="preserve">
+    <value>NewMethod</value>
+  </data>
+  <data name="ExtractParameterNewType_ByRefParameter" xml:space="preserve">
+    <value>Parameter passed by reference</value>
+  </data>
+  <data name="ExtractParameterNewType_ByValParameter" xml:space="preserve">
+    <value>Parameter passed by value</value>
+  </data>
+  <data name="ExtractMethod_ParameterAssignmentTypeColumnHeader" xml:space="preserve">
+    <value>Pass by type</value>
+  </data>
+  <data name="ExtractMethod_ParameterDataTypeColumnHeader" xml:space="preserve">
+    <value>Data type</value>
+  </data>
+  <data name="ExtractMethod_ParameterNameColumnHeader" xml:space="preserve">
+    <value>Name</value>
+  </data>
+  <data name="ExtractMethod_UnableToMoveVariableDeclarationMessage" xml:space="preserve">
+    <value>Unable to move variable declaration</value>
+  </data>
+  <data name="ExtractMethod_InvalidMessageSelectionEndsOutsideProcedure" xml:space="preserve">
+    <value>Selection does not end inside a recognised procedure</value>
+  </data>
+  <data name="ExtractMethod_InvalidMessageSelectionHasEndStatement" xml:space="preserve">
+    <value>Extract method cannot extract code that contains an End statement.</value>
```

Comment:
Would probably be easier for translators if it were just one resource key with a format-placeholder, like "Extract Method cannot extract code that contains a '{0}' token."